packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -244,7 +244,7 @@ export const VAutocomplete = genericComponent<new <
         const originalSelectionIndex = selectionIndex.value
 
         const selectedItem = model.value[selectionIndex.value]
-        if (selectedItem) select(selectedItem)
+        if (selectedItem && !selectedItem.props?.disabled) select(selectedItem)
```

Comment:
Why optional chaining? Props is always defined.

Reply:
should be detected with `@typescript-eslint/no-unnecessary-condition` but this rule seems to be turned off